### PR TITLE
Fix test args for regional tests

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -7126,7 +7126,7 @@
       "--gke-command-group=beta",
       "--gke-environment=test",
       "--provider=gke",
-      "--test_args=--gce-multizone=true --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
+      "--test_args=--gce-multizone=true --gce-region=us-central1 --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
```
Comment (shyamjvs) [inline jobs/config.json]: Isn't it gcp-region?

Comment (wojtek-t) [reply jobs/config.json]: No.
--gcp-region is a flag to kubetest (that is not propagated to test binary)

--gce-region is the flag of e2e binary

Comment (shyamjvs) [reply jobs/config.json]: Aah.. right. It's part of test-args.
That was confusing.

**Comment by shyamjvs** [comment]
@krzyzacy @BenTheElder Seems like we shouldn't need to set --gcp-region and also pass --gce-region separately in test-args. Could you unite both of those (i.e set --gce-region in test-args based on --gcp-region)?

**Comment by krzyzacy** [comment]
isn't e2e framework reads $REGION?

**Comment by wojtek-t** [comment]
> isn't e2e framework reads $REGION?

Not now (I don't know maybe it did in the past).

**Pull request merged**